### DISCUSSION TITLE
[Mobile] Maccatalyst can't use dotnet run dll

### DISCUSSION THREAD

**Discussion by lld1995** [unoplatform/uno#14293] [Q&A] [answered]
### Current behavior

I use dotnet MyApp.Mobile.dll ,but it shows error:
A fatal error was encountered. The library 'libhostpolicy.dylib' required to execute the application was not found in '/Users/west/Documents/MyApp/MyApp.Mobile/bin/Debug/net7.0-maccatalyst/maccatalyst-x64/'.
Failed to run as a self-contained app.
  - The application was run as a self-contained app because '/Users/west/Documents/MyApp/MyApp.Mobile/bin/Debug/net7.0-maccatalyst/maccatalyst-x64/MyApp.Mobile.runtimeconfig.json' did not specify a framework.
  - If this should be a framework-dependent app, specify the appropriate framework in '/Users/west/Documents/MyApp/MyApp.Mobile/bin/Debug/net7.0-maccatalyst/maccatalyst-x64/MyApp.Mobile.runtimeconfig.json'.

### Expected behavior

Runs like win or linux

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report.

Catalyst apps are packaged apps in the same sense that iOS apps are packaged. They don't start like any other CLI apps. See [this article](https://learn.microsoft.com/en-us/dotnet/maui/mac-catalyst/deployment/publish-ad-hoc) or [this article](https://learn.microsoft.com/en-us/dotnet/maui/mac-catalyst/deployment/publish-unsigned) on the .NET documentation.
